src/Majora/Bundle/OAuthServerBundle/Security/AccessTokenAuthenticator.php

```diff
@@ -53,7 +53,8 @@ public function getCredentials(Request $request)
 
             // token through headers
             case $request->headers->has('Authorization') :
-                if (!preg_match('#^Bearer ([\w]+)$#', $request->headers->get('Authorization'), $matches)) {
+                $accessTokenPattern = '[\w--_]+';
```

Comment:
your regexp seems to allow more characters than it should
why not `[\w-_\.]+` instead ?